packages/lib/src/components/core/lume-axis/composables/lume-skip.ts

```diff
@@ -9,12 +10,12 @@ const SAFETY_MARGIN = 0.2; // 20% of the average width
  *
  * @param scale A d3 scale.
  * @param tickRefs Template reference to the tick text elements
- * @param skipNumber Optional amount of ticks to skip
+ * @param options The axis options.
  */
 export function useSkip(
   scale: Ref<Scale>,
   tickRefs: Ref<Array<{ ref: SVGTextElement }>>,
-  skipNumber?: true | number
+  options: Ref<AxisOptions>
```

Comment:
that was a great catch!
non-reactive as we were not passing the composable with a reactive value 🤯